### PR TITLE
Adding ttH ctcvcp cards

### PR DESCRIPTION
**Pull request by stiegerb**
ttH sample with violating CP phase in the t-H coupling, to be used as background for tHq analysis.

### CODE REVIEW

**Comment by chayanit** [comment]
@perrozzi @efeyazgan could you check and merge this PR? thanks

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Higgs/tth_4f_ckm_LO_ctcvcp/tth_4f_ckm_LO_ctcvcp_madspin_card.dat [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+#*                                                          *
+#************************************************************
+#Some options (uncomment to apply)
+#
+#directory for gridpack mode
+set ms_dir ./madspingrid
+
+#initialization parameters
+ set Nevents_for_max_weigth 250 # number of events for the estimate of the max. weight
+ #set BW_cut 15                # cut on how far the particle can be off-shell
```
Comment (efeyazgan) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Higgs/tth_4f_ckm_LO_ctcvcp/tth_4f_ckm_LO_ctcvcp_madspin_card.dat]: Please delete this commented input line

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Higgs/tth_4f_ckm_LO_ctcvcp/tth_4f_ckm_LO_ctcvcp_madspin_card.dat [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+define l- = e- mu- ta-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+define wdec = u c d s u~ c~ d~ s~ l+ l- vl vl~
+
+# specify the decay for the final state particles
+#(inclusive top decays in this case)
+decay t > w+ b, w+ > wdec wdec
+decay t~ > w- b~, w- > wdec wdec
+#decay w+ > l+ vl
```
Comment (efeyazgan) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Higgs/tth_4f_ckm_LO_ctcvcp/tth_4f_ckm_LO_ctcvcp_madspin_card.dat]: Please delete this line and the line below.

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Higgs/tth_4f_ckm_LO_ctcvcp/tth_4f_ckm_LO_ctcvcp_proc_card.dat [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+#*                                                          *
+#*     run as ./bin/mg5_aMC  filename                       *
+#*                                                          *
+#************************************************************
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model sm
+define p = g u c d s u~ c~ d~ s~
```
Comment (efeyazgan) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Higgs/tth_4f_ckm_LO_ctcvcp/tth_4f_ckm_LO_ctcvcp_proc_card.dat]: remove p and j definitions.

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Higgs/tth_4f_ckm_LO_ctcvcp/tth_4f_ckm_LO_ctcvcp_run_card.dat [hunk 4]
```diff
@@ -0,0 +1,273 @@
...
+  6500.0	=  ebeam2   ! beam 2 total energy in GeV
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+  0.0	=  polbeam1  ! beam polarization for beam 1
+  0.0	=  polbeam2  ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+  lhapdf	=  pdlabel      ! PDF set
```
Comment (efeyazgan) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Higgs/tth_4f_ckm_LO_ctcvcp/tth_4f_ckm_LO_ctcvcp_run_card.dat]: Please use
$DEFAULT_PDF_SETS = lhaid
$DEFAULT_PDF_MEMBERS = reweight_PDF

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Higgs/tth_4f_ckm_LO_ctcvcp/tth_4f_ckm_LO_ctcvcp_run_card.dat [hunk 5]
```diff
@@ -0,0 +1,273 @@
...
+#
+#*********************************************************************
+# Store info for systematics studies                                 *
+# WARNING: If use_syst is T, matched Pythia output is                *
+#          meaningful ONLY if plotted taking matchscale              *
+#          reweighting into account!                                 *
+#*********************************************************************
+  True	=  use_syst       ! Enable systematics studies
+#
+#**************************************
```
Comment (efeyazgan) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/Higgs/tth_4f_ckm_LO_ctcvcp/tth_4f_ckm_LO_ctcvcp_run_card.dat]: Delete these comments for syscalc

**Comment by efeyazgan** [comment]
I added some comments inline for the cards (following https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T)

**Comment by stiegerb** [comment]
Hi @efeyazgan 
Thanks, I updated the cards. Please have another look.
(I wasn't aware of those instructions, sorry.)

Cheers

**Comment by efeyazgan** [comment]
No problem. Thanks for the updated cards.

**Pull request merged**